src/unitxt/metrics.py

```diff
@@ -12,6 +12,7 @@
 import evaluate
 import numpy
 import numpy as np
+import pytrec_eval
```

Comment:
This import should move inside the  'pytrec_eval_at_k' function so it will not be required by people not using the metric.